src/EditorFeatures/TestUtilities/Workspaces/TestHostProject.cs

```diff
@@ -364,7 +364,8 @@ public ProjectInfo ToProjectInfo()
                 this.IsSubmission,
                 this.HostObjectType,
                 hasAllInformation: true,
-                runAnalyzers: true)
+                runAnalyzers: true,
+                telemetryId: default)
```

Comment:
Ditto to other question: should this Create overload just have some default parameters?